packages/main/src/sync.ts

```diff
@@ -337,20 +338,25 @@ class BoxSync {
 }
 
 function matchesCachePolicy(message: imap.ImapMessageAttributes): boolean {
+  let labelMatches = true
```

Comment:
Please declare these variables with `const`. You can collapse the logic below into single RHS expressions.